include/net/net_pkt.h

```diff
@@ -460,18 +460,33 @@ static inline void net_pkt_set_src_ipv6_addr(struct net_pkt *pkt)
 /* @endcond */
 
 /**
- * @brief Create a TX net_pkt slab that is used when sending user
- * specified data to network.
+ * @brief Create a net_pkt slab
  *
- * @param name Name of the pool.
+ * A net_pkt slab is used to store (meta)information about
```

Comment:
This should just be "metainformation" or better would be "metadata"

Reply:
Well, I'd prefer to have this meta in parens to avoid it sounding like a real term. But the purpose of "meta" is indeed to emphasize the difference from net_buf's, which store packet payload data.

Reply:
not sure what you mean here.  "metadata" is a real term?

Reply:
Yes, it's a term with a specific meaning (meanings!): https://en.wikipedia.org/wiki/Metadata . I actually never heard term "metadata" to be used for network packet information, and would like to avoid establishing that as a term in Zephyr. The only reason I use "meta" prefix is to emphasize contrast with net_buf's which contain *payload* data (while net_pkt's - don't). Just writing "net_pkt contains network packet information" doesn't emphasize the different enough IMHO.

Summary: seems like my attempt to deconfuse why there's a need for 2 types of buffers actually leads to more confusion. Let's see of @jukkar / @tbursztyka will like that "meta" thingy at all, if not, I'll just drop it.